.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-    - "0.12"
-    - "0.11"
-    - "0.10"
+    - "4.1"
+    - "4.0"
```

Comment:
Since `protractor v3.x` uses the new Selenium webdriver, harmony support is required. The protractor team no longer supports old node versions with their `3.0.0` release https://github.com/angular/protractor/commit/18e1f71a4dd868709f4e259e05a8a196921e22be

We can add node `v5` support to these tests as well, but unfortunately we cannot support older node versions with protractor `v3+`